CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 3.31.0
+### Added
+ * Integration packages to instrument AI model invocations (see below).
+    * New package nrawsbedrock v1.0.0 introduced to instrument calls to Amazon Bedrock Runtime Client API `InvokeModel` and `InvokeModelWithResponseStream` calls.
+    * New package nropenai v1.0.0 introduced to instrument OpenAI model invocations. 
```

Comment:
How about "New package nropenai v1.0.0 introduced to instrument calls to OpenAI API using `NRCreateChatCompletion`, `NRCreateChatCompletionStream`, and `NRCreateEmbedding` calls"?